gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# PR for #143: Create Squad Stack

### Description

In the event stack, squads should be shown, not individual user. 
Matches.html NOT yet completed. 

### Testing Directions

```
./setup
python manage runserver
```

1.  Create a squad with several users.
2.  Have at least one of those users marked "interested" or "going" to an event.
3.  Have another user, user2, not in the squad mark "interested" or "going" to the same event.
4.  User2 should be able to see the squad in the event stack. 

### Checklist
